src/style.css

```diff
@@ -169,3 +175,51 @@ i.fa {
     from { transform: scale(1) rotate(0deg);}
     to { transform: scale(1) rotate(360deg);}
 }
+
+.doc-page {
```

Comment:
I think it's time we consider using sass to build our css and break it into several source files. But we can worry about that after this PR gets merged.